### PR TITLE
Added 5 columns to the sessions / export csv

### PR DESCRIPTION
**Pull request by cabch**
To specify the timezone and time of the charging station only in the exported csv file

### CODE REVIEW

File: src/server/rest/v1/service/TransactionService.ts [hunk 1]
```diff
@@ -966,7 +966,7 @@ export default class TransactionService {
         projectFields = [
           ...projectFields,
           'userID', 'user.id', 'user.name', 'user.firstName', 'user.email',
-          'stop.userID', 'stop.user.id', 'stop.user.name', 'stop.user.firstName', 'stop.user.email'
+          'stop.userID', 'stop.user.id', 'stop.user.name', 'stop.user.firstName', 'stop.user.email',
```
Comment (ClaudeROSSI) [inline src/server/rest/v1/service/TransactionService.ts]: ??? why are you putting back that comma character at the end?

Comment (LucasBrazi06) [reply src/server/rest/v1/service/TransactionService.ts]: To easily add new entries :-)

**Review by ClaudeROSSI — APPROVED** [review]

**Review by ClaudeROSSI — APPROVED** [review]

File: src/server/rest/v1/service/TransactionService.ts [hunk 2]
```diff
@@ -857,10 +857,11 @@ export default class TransactionService {
       csv += `${transaction.connectorId}` + Constants.CSV_SEPARATOR;
       csv += `${transaction.user ? Cypher.hash(transaction.user.id) : ''}` + Constants.CSV_SEPARATOR;
       csv += `${transaction.user ? Utils.buildUserFullName(transaction.user, false) : ''}` + Constants.CSV_SEPARATOR;
-      csv += `${moment(transaction.timestamp).format('YYYY-MM-DD')}` + Constants.CSV_SEPARATOR;
-      csv += `${moment(transaction.timestamp).format('HH:mm:ss')}` + Constants.CSV_SEPARATOR;
-      csv += `${transaction.stop ? `${moment(transaction.stop.timestamp).format('YYYY-MM-DD')}` : ''}` + Constants.CSV_SEPARATOR;
-      csv += `${transaction.stop ? `${moment(transaction.stop.timestamp).format('HH:mm:ss')}` : ''}` + Constants.CSV_SEPARATOR;
+      csv += `${transaction.timezone}` + Constants.CSV_SEPARATOR;
```
Comment (LucasBrazi06) [inline src/server/rest/v1/service/TransactionService.ts:860]: The timezone may not be provided in transaction if the charging station does not have GPS coordinates.
Check if the timezone (tz) is provided and provide a default behavior with an empty tz value:
```
      csv += `${moment(transaction.timestamp).format('YYYY-MM-DD')}` + Constants.CSV_SEPARATOR;
      csv += `${moment(transaction.timestamp).format('HH:mm:ss')}` + Constants.CSV_SEPARATOR;
      csv += `${transaction.stop ? `${moment(transaction.stop.timestamp).format('YYYY-MM-DD')}` : ''}` + Constants.CSV_SEPARATOR;
      csv += `${transaction.stop ? `${moment(transaction.stop.timestamp).format('HH:mm:ss')}` : ''}` + Constants.CSV_SEPARATOR;
```

Comment (cabch) [reply src/server/rest/v1/service/TransactionService.ts:860]: Ok I didn't know the tz can be missing. Going to fix it

Comment (LucasBrazi06) [reply src/server/rest/v1/service/TransactionService.ts:860]: Yes, no problem!
I just checked the code while reviewing, forgot to mention this to you!

File: src/server/rest/v1/service/TransactionService.ts [hunk 3]
```diff
@@ -857,11 +857,11 @@ export default class TransactionService {
...
-      csv += `${moment(transaction.timestamp).tz(transaction.timezone).format('HH:mm:ss')}` + Constants.CSV_SEPARATOR;
-      csv += `${transaction.stop ? `${moment(transaction.stop.timestamp).tz(transaction.timezone).format('YYYY-MM-DD')}` : ''}` + Constants.CSV_SEPARATOR;
-      csv += `${transaction.stop ? `${moment(transaction.stop.timestamp).tz(transaction.timezone).format('HH:mm:ss')}` : ''}` + Constants.CSV_SEPARATOR;
+      csv += `${transaction.timezone || 'N/A (UTC by default)'}` + Constants.CSV_SEPARATOR;
+      csv += `${(transaction.timezone ? moment(transaction.timestamp).tz(transaction.timezone) : moment.utc(transaction.timestamp)).format('YYYY-MM-DD')}` + Constants.CSV_SEPARATOR;
+      csv += `${(transaction.timezone ? moment(transaction.timestamp).tz(transaction.timezone) : moment.utc(transaction.timestamp)).format('HH:mm:ss')}` + Constants.CSV_SEPARATOR;
+      csv += `${transaction.stop ? `${(transaction.timezone ? moment(transaction.stop.timestamp).tz(transaction.timezone) : moment.utc(transaction.stop.timestamp)).format('YYYY-MM-DD')}` : ''}` + Constants.CSV_SEPARATOR;
```
Comment (ClaudeROSSI) [inline src/server/rest/v1/service/TransactionService.ts:863]: From my perspective the back tick (template strings) are useless here.
especially the one which are nested.

Can't we just do it like this:
 csv += transaction.id + Constants.CSV_SEPARATOR;

Comment (LucasBrazi06) [reply src/server/rest/v1/service/TransactionService.ts:863]: Well I love back quotes :)
But in this example where we have:
`      csv += `${transaction.id}` + Constants.CSV_SEPARATOR;`
You're right, back quotes are useless!

Comment (cabch) [reply src/server/rest/v1/service/TransactionService.ts:863]: So, do we delete all back quotes ? every line for the headers (in this file but in others too) can work as well without (we tried ;))

Comment (LucasBrazi06) [reply src/server/rest/v1/service/TransactionService.ts:863]: To keep consistency, if we remove it for few of them, let's also remove it for all CSV properties here.

Comment (cabch) [reply src/server/rest/v1/service/TransactionService.ts:863]: yes I will delete every back quotes for all the csv stuff in every files I find csv :)

**Review by LucasBrazi06 — APPROVED** [review]
PR Approved, backticks will be changed in the next one as discussed with Caro.

**Pull request merged**